packages/apps/wallet/src/pages/Wallet/index.tsx

```diff
@@ -604,7 +604,7 @@ const WalletPage = () => {
                                 alignContent: 'flex-end',
                               }}
                               gap={8}
-                            >
+                            >{console.log(collectionData)}
```

Comment:
Please remove the console.log statement.

Reply:
done, forgot about it